conda_smithy/feedstock_content/.gitignore

```diff
@@ -8,7 +8,7 @@
 !/conda-forge.yml
 
 # Don't ignore any subfolders if the parent folder is 'un-ignored'
```

Comment:
@isuruf, WDYT about the following?
```suggestion
# This avoids warnings for `git add`ing already checked-in files in otherwise ignored folders.
```

Reply:
That's wrong. It's not about warnings for already checked-in files. Without this line, adding `recipe/new_file.txt` will not work without a force push.

Reply:
alright, I see you added this in https://github.com/conda-forge/conda-smithy/pull/1822/commits/8ec20693bba630f6d820d90416094bd80101d273 after all.